### ISSUE TITLE
Update RP2 wiki: Hack for LA volunteers transferrable skills

### ISSUE THREAD

**Issue by adarosh** [hackforla/internship#297]
### Overview
We need to review and update the wiki page for RP2: Hack for LA volunteers transferrable skills so that future HfLA volunteers can replicate our processes.

### Action Items
- [x] Add milestone: 06.01.02 Research Audit and update wiki page
- [x] Review/Update Research Date
- [x] Review/Update Summary of Research
    - [x] Update Research Question
    - [x] Update Purpose
    - [x] Update Methods
- [x] Review/Update Current Status
- [x] Search all issues to make sure that there are no mislabeled issues. 
- [x] Make sure that the "All Issues" link correctly identifies all issues related to this RP (both open and closed)
- [x] Review/Update Outstanding task items
- [x] Review/Update Assets
    - [x] Review/Update link to Research Plan
    - [x] Review/Update link to Scripts Docs
    - [x] If other relevant subheadings need to be created, create those headings (using RP6 as an example)
    - [x] Review/update link to Interview Recordings & Transcripts (or relabel if it's quantitative data)
    - [x] Review/update link to Synthesis (if nothing to report yet, leave blank)
    - [x] Review/update link to Presentation of Findings (if nothing to report yet, leave blank)
    - [x] Review/update link to Action Items Spreadsheet (if nothing to report yet, leave blank)
- [x] Review/Update Links to Related Wiki Pages
- [x]  Research lead reviews wiki page
- [ ]  Product reviews wiki page

### Resources/Instructions
- [Research Plan 2: Hack for LA volunteers transferrable skills](https://github.com/hackforla/internship/wiki/Research-Plan-2:-Hack-for-LA-volunteers-transferrable-skills)
- [Example Wiki Research Plan Page: RP6](https://github.com/hackforla/internship/wiki/Research-Plan-6:-Intern-Intake-Interviews)
- [Google Drive, Research Folder](https://drive.google.com/drive/folders/1LLHYcQCRmpC3DX_ZMj4BXNCmb_XliUXd?usp=sharing) - you can find research  plans by type or by number/name
- Don't link GitHub issues, instead make sure that the wiki page has a link to all the issues using the label (e.g., `Research: RP006` label)
- If a particular Research Plan was not conducted during the Internship 2022, note why under "Current Status".
- Tips for identifying mislabeled GitHub issues
    - In the search bar, type keywords that might be related to this research plan
    - Start broad to identify keywords that might indicate the same thing. Quickly scroll through all issues to see if the language to describe this research plan was inconsistent (e.g., "intake" vs. "onboarding"). Make sure that no matter the language that the RP label is correct.
    - Look in both open and closed issues
- [Example of action items spreadsheet](https://docs.google.com/spreadsheets/d/1WGim70shHIt-6BbweDvbn-J0IdOR8JmQLYg9Rovf7bA/edit#gid=1693908723)

**Comment by ExperimentsInHonesty** [comment]
@alaishasayed Please provide update
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures or links* (if necessary): "Add any pictures or links that will help illustrate what you are working on."

* remember to add links to the top of the issue if they are going to be needed again.

**Comment by alaishasayed** [comment]
1. Progress: 
2. Blockers: 
3. Availability: 
4. ETA: 
5. Pictures or links (if necessary):

**Comment by alaishasayed** [comment]
Progress: complete
Blockers: some documents have not been completed
Availability: no more needed, will address new issue
ETA: done
Pictures or links (if necessary):

**Comment by alaishasayed** [comment]
Interview script, questions, and form have been created, but not yet administered yet. Participants have not been recruited.

**Comment by bradyse** [comment]
@alaishasayed 

Thanks for the progress on this issue. Please make sure that you complete the following, as you check off the remaining items off this issue:

- If a particular Research Plan was not conducted during the Internship 2022, note why under "Current Status".
- Tips for identifying mislabeled GitHub issues
     - In the search bar, type keywords that might be related to this research plan
     - Start broad to identify keywords that might indicate the same thing. Quickly scroll through all issues to see if the language to describe this research plan was inconsistent (e.g., "intake" vs. "onboarding"). Make sure that no matter the language that the RP label is correct.
     - Look in both open and closed issues

Once all the boxes are checked off, make a comment again in this issue to let me know and then you can pick up the next issue.

**Comment by alaishasayed** [comment]
A related issue was located - [Review Interview Instructions for Research Plan 2: find transferable skills #50
](https://github.com/hackforla/internship/issues/50) - to review the instructions of the interview questions, however it appears as if this task was not yet completed. This was put "in the icebox with the dependency for the script questions to be finalized" 

@bradyse

**Comment by bradyse** [comment]
@alaishasayed 

Find related research assets in #50 and put those assets in RP2's Google Drive folder

**Comment by alaishasayed** [comment]
Progress: almost complete! Found a related page in icebox with "dependency" label.  This page shared that instructions of interview questions needed to be reviewed and finalized, but never were.  Moved the asset related to this page to RP2 Google Drive.
Blockers: Interview Instructions were never finalized which is why interviews were never conducted.  It appears as though preparation for interviews still needed to be done such as finalizing instructions and recruiting participants.
Availability: this weekend 
ETA: hopefully next Thursday. 
Pictures or links (if necessary): [Interview Instructions Google Doc](https://docs.google.com/document/d/1gKklWCPKtVQBbsYoLi59t7tgVKu0gb1MMyyh9qk0qIs/edit)

**Comment by bradyse** [comment]
@ExperimentsInHonesty @priyatalwar @sharongupta1 

The UXR team isn't sure what is an "Action Item Spreadsheet" and why that is included on the wiki page template. What is that? However, all the major wiki items have been updated and this issue is complete and can be closed.

**Comment by sharongupta1** [comment]
@bradyse Please add the "Action Item Spreadsheet" link to the Resources section above

**Comment by priyatalwar** [comment]
@pandanista @bradyse  I reviewed the wiki page and the link to the actions items spreadsheet is missing. Can you please update the wiki with that link? Thanks!

**Comment by pandanista** [comment]
Hi @priyatalwar, I don't think we have used action item spreadsheet for internship's wiki as far as I know. Instead, we've been listing the action items under "Outstanding task items" section on wiki. 
<img width="708" alt="Screenshot 2023-07-20 at 1 48 05 PM" src="https://github.com/hackforla/internship/assets/57029070/1e0bff99-2303-4643-ba93-40b03e32cd68">

If the "Outstanding task items" section meets Product's needs, please go ahead and close the issue.

If we are not using action item spreadsheet, we should also update the wiki template to avoid any future confusion.

**Comment by ExperimentsInHonesty** [comment]
The action items spreadsheet is for when the analysis is done, and there is recomendations for the team to review and integrate.  

The outstanding task items is for research work on the RP that still needs to be done prior to presenting the action items spreadsheet.
